mico-core/src/test/java/io/github/ust/mico/core/DeploymentControllerIntegrationTests.java

```diff
@@ -171,7 +164,7 @@ private MicoApplication getTestApplication(MicoService service) {
             .setShortName("hello")
             .setName("hello-application")
             .setVersion("v1.0.0");
-            // TODO Refactor Deployment info (redundant information in comparison with MicoService)
+        // TODO Refactor Deployment info (redundant information in comparison with MicoService)
             /*.deploymentInfo(MicoApplicationDeploymentInfo.builder()
```

Comment:
remove commented code

Reply:
That would make sense in context with #288 -> @JakobBenz

Reply:
Any commented code is useless in the master. Therefore: remove it.

Reply:
I agree with you, that was a mistake I made in an older pull requests.
Jakob already made the clean up in the branch `feature/mico#288` I'm currently reviewing.